operators/pkg/forge/ingresses_test.go

```diff
@@ -270,38 +327,103 @@ var _ = Describe("Ingresses", func() {
 			)
 		})
 
-		Describe("The forge.IngressInstancePath function", func() {
-			JustBeforeEach(func() {
-				path = forge.IngressInstancePath(&instance)
+		Describe("The forge.IngressGUIPath function", func() {
+
+			When("EnvironmentType is ClassStandalone", func() {
+
+				When("Rewrite is true", func() {
+					BeforeEach(func() {
+						environment.EnvironmentType = clv1alpha2.ClassStandalone
+						environment.RewriteURL = true
+					})
+					JustBeforeEach(func() {
+						path = forge.IngressGUIPath(&instance, &environment)
+					})
```

Comment:
Could be moved a level above to reduce code duplication.

Reply:
done